packages/js/components/src/experimental-select-control/selected-items.tsx

```diff
@@ -22,14 +24,34 @@ type SelectedItemsProps< ItemType > = {
 };
 
 export const SelectedItems = < ItemType, >( {
+	isReadOnly,
 	items,
 	getItemLabel,
 	getItemValue,
 	getSelectedItemProps,
 	onRemove,
 }: SelectedItemsProps< ItemType > ) => {
+	const classes = classnames(
+		'woocommerce-experimental-select-control__selected-items',
+		{
+			'is-read-only': isReadOnly,
+		}
+	);
+
+	if ( isReadOnly ) {
```

Comment:
Could we make this not the default? and allow us to have an additional `prop` that either enables the comma separated list, or more abstract and expose a call back where a component can render the items manually:
```
selectedItems: ( ...props ) => false | JSX.Element; // if false we render the normal?
```

Reply:
I mostly mention this, as we use this within Attributes as well, and the comma separated items looks weird when a placeholder is also set:
![Screenshot 2023-05-01 at 12 27 12 PM](https://user-images.githubusercontent.com/2240960/235441628-1792a983-9130-422e-b520-94d8f9f01690.png)
@jarekmorawski I don't know if your intention is to have this to be the new default for all multi-select dropdowns?

Reply:
Yes, although we may want to do another iteration based on user feedback. 👍 No need to show the placeholder text when the field is not focused and the dropdown is closed.

Reply:
Updated this to hide the placeholder when in read only mode.  Kept the comma separation for all experimental select controls.